include/openenclave/edl/time.edl

```diff
@@ -29,5 +29,12 @@ enclave
             [in] struct oe_timespec* req,
             [in, out] struct oe_timespec* rem)
             propagate_errno;
+
+        int oe_syscall_clock_nanosleep_ocall(
+            int clockid,
```

Comment:
@mingweishih clockid is defined as clockid_t which is inturn an int (typedef clockid_t int) in the linux implementation. Is it okay to keep it as int here or should I include <time.h> and change type to clockid_t?

Reply:
Can you define it in the openenclave/bints/types.h?